tests/test_suite/run_test_command.py

```diff
@@ -767,3 +818,66 @@ def otx_train_auto_config(root, otx_dir: str, args: Dict[str, str]):
     command_line.extend(["--work-dir", f"{work_dir}"])
     command_line.extend(args["train_params"])
     check_run(command_line)
+
+
+def otx_eval_compare(
+    template,
+    root,
+    otx_dir,
+    args,
+    criteria,
+    result_dict,
+    threshold=0.10,
+):
+    template_work_dir = get_template_dir(template, root)
+
+    command_line = [
+        "otx",
+        "eval",
+        template.model_template_path,
+        "--test-data-roots",
+        f'{os.path.join(otx_dir, args["--test-data-roots"])}',
+        "--load-weights",
+        f"{template_work_dir}/trained_{template.model_template_id}/weights.pth",
+        "--save-performance",
+        f"{template_work_dir}/trained_{template.model_template_id}/performance.json",
+    ]
+    command_line.extend(["--work-dir", f"{template_work_dir}"])
+    command_line.extend(args.get("eval_params", []))
+    check_run(command_line)
+
+    performance_json_path = f"{template_work_dir}/trained_{template.model_template_id}/performance.json"
+    assert os.path.exists(performance_json_path)
+
+    with open(performance_json_path) as read_file:
+        trained_performance = json.load(read_file)
+
+    model_criteria = criteria[template.name]
+    modified_criteria = model_criteria - (model_criteria * threshold)
+    for k in trained_performance.keys():
+        result_dict[k] = round(trained_performance[k], 3)
+        assert (
+            trained_performance[k] >= modified_criteria
+        ), f"Current model performance: ({trained_performance[k]}) < criteria: ({modified_criteria})."
+
+    result_dict["Model size (MB)"] = round(
+        os.path.getsize(f"{template_work_dir}/trained_{template.model_template_id}/weights.pth") / 1e6, 2
+    )
+
+
+def otx_eval_e2e_train_time(train_time_criteria, e2e_train_time, template, threshold=0.10):
```

Comment:
Can you add a docstring to these methods to explain these tests. I am not familiar with them so I am not aware of what is `train_time_criteria` and `e2e_train_time`

Reply:
Done